configure.ac

```diff
@@ -571,6 +571,53 @@ if test "X$xslt_config" != "Xno" ; then
   fi
 fi
 
+libxslt_source=no
```

Comment:
I think libxslt_source is never changed so the code below (and this line) are redundant

Reply:
`libxslt_source` was changed in the 'action if present' part of the PKG_CHECK_MODULES call. The code sequence serves not useful purpose, so I removed it.